browser_utils.js

```diff
@@ -400,9 +453,14 @@ async function clickText(page, text, {timeout=30000, checkEvery=200, elementXPat
 }
 
 /**
- * @param {import('puppeteer').Page} page 
- * @param {string} testId 
- * @param {{extraMessage?: string, timeout?: number, visible?: boolean}} [options] 
+ * Click an element identified by a test ID (`data-testid=` attribute).
+ * Selecting and clicking happens in the same tick, so this is safe to call even if the client application may currently be rerendering.
+ *
+ * @param {import('puppeteer').Page} page The puppeteer page handle.
+ * @param {string} testId The test ID to look for.
+ * @param {{extraMessage?: string, timeout?: number, visible?: boolean}} [__namedParameters] Options (currently not visible in output due to typedoc bug)
+ * @param {string?} extraMessage Error message shown if the element is not present in time.
+ * @param {number?} timeout How long to wait, in milliseconds.
```

Comment:
2 more defaults

Reply:
`extraMessage` is optional, so no default there.